extensions/amp-story/1.0/_locales/de.js

```diff
@@ -89,10 +89,10 @@ export default {
         'sehen',
   },
   [LocalizedStringId.AMP_STORY_WARNING_LANDSCAPE_ORIENTATION_TEXT]: {
-    string: 'Diese Seite is am Besten im Portrait Mode',
+    string: 'Diese Seite sieht am Besten im "Hochformat" aus',
   },
   [LocalizedStringId.AMP_STORY_WARNING_UNSUPPORTED_BROWSER_TEXT]: {
-    string: 'Es tut uns sehr leid, aber es sieht so aus als dass der Browser ' +
+    string: 'Es tut uns sehr leid, aber es sieht so aus als wenn der Browser ' +
```

Comment:
"als wenn" also seems colloquial, no? Even besides that ... this translation is quite awkward overall :) What do you think about: "Es tut uns leid, aber Ihr Browser unterstützt diese Format leider nicht."?

Reply:
"Es tut uns leid, aber Ihr Browser unterstützt diese Format leider nicht." is definitely the better translation.
Yes, let's take that one.